examples/pdc_test_cache.rb

```diff
@@ -47,3 +47,6 @@ def main
 end
 
 main if __FILE__ == $PROGRAM_NAME
+
+a = 'aaaaa'
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `a`. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)